src/scripts/main.js

```diff
@@ -1,3 +1,46 @@
 'use strict';
 
-// write code here
+const title = document.querySelectorAll('th');
+
+for (let i = 0; i < 3; i++) {
+  title[i].onclick = () => {
+    const tBodyChildren = Array.from(document.querySelector('tbody').children);
+
+    tBodyChildren.sort((a, b) => {
+      const num1 = a.children[i].innerText;
+      const num2 = b.children[i].innerText;
+
+      if (num1 > num2) {
+        return 1;
+      }
+
+      if (num1 < num2) {
+        return -1;
+      }
+
+      return 0;
```

Comment:
```suggestion
      return num1 - num2
```
Можна було просто повертати різницю в цьому колбеку